federation/cmd/federation-controller-manager/app/controllermanager.go

```diff
@@ -200,13 +200,10 @@ func StartControllers(s *options.CMServer, restClientCfg *restclient.Config) err
 		ingressController.Run(wait.NeverStop)
 	}
 
-	glog.Infof("Loading client config for service controller %q", servicecontroller.UserAgentName)
-	scClientset := federationclientset.NewForConfigOrDie(restclient.AddUserAgent(restClientCfg, servicecontroller.UserAgentName))
-	servicecontroller := servicecontroller.New(scClientset, dns, s.FederationName, s.ServiceDnsSuffix, s.ZoneName, s.ZoneID)
+	servicecontrollerClientset := federationclientset.NewForConfigOrDie(restclient.AddUserAgent(restClientCfg, "service-controller"))
```

Comment:
Sorry i missed and a mistake. while refactoring i did not do line by line comparison, just replaced the functionality by using new federated_informer framework.
will check again and add back any logs missed.